rules/network/command_and_control_fin7_c2_behavior.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2020/07/06"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects a known command and control pattern in network events. The FIN7 threat group is known to use this
```

Comment:
I was tempted to say remove _This rule_ from `description` and `false_positives`, but then I realized we have 22 rules where `description` starts this way. It does seem a bit unnecessary IMO, but if we are consistent either way, I think that is best (meaning if we remove here, we do everywhere). We can discuss this before deciding.

Reply:
As an author and operator, I like the BLUF consistency of "this rule does this thing" and then adds some context vs. having to read through a paragraph each to ascertain what I think it means.

If we want to remove it, that's fine, but we should replace it with something we can "standardize".

Reply:
Agreed. When triaging, conciseness goes a long way and I was simply commenting that with `This rule detects ... things` vs `Detects ... things` that it is implied the description is referring to the rule. Your call either way 👍

Reply:
Thanks, @brokensound77 great insight.

I think we should look at standardizing descriptions, false positives, notes, etc and update existing rules once we've landed on a solution.

Reply:
Agree with this. A style guide of rule writing would be helpful. Even standardizing on the ordering of search terms would be helpful to offer a common convention to make logic easier to understand across the entire ruleset.

Reply:
We do have some verbiage on query ordering: https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md#using-elastic-common-schema-ecs 

As well as writing style: https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md#writing-style

But there is def room for standardization, which may evolve organically over time